packages/lightning-lsp-common/src/resources/sfdx/typings/lds.d.ts

```diff
@@ -632,6 +682,17 @@ declare module 'lightning/analyticsWaveApi' {
         url: string;
     }
 
+    /**
+     * A Connection Property Name and Value.
+     *
```

Comment:
We should add the link to ConnectionPropertyValueRepresentation here to be consistent: https://developer.salesforce.com/docs/atlas.en-us.bi_dev_guide_rest.meta/bi_dev_guide_rest/bi_resources_dataconnectors.htm#ConnectionPropertyValueRepresentation